pkg/reporter/sync.go

```diff
@@ -516,6 +516,24 @@ func (s *syncReporter) syncPackage(pkg *models.Package) error {
 		})
 	}
 
+	// Add malware analysis information if available
+	if mar := pkg.GetMalwareAnalysisResult(); mar != nil {
+		req.MaliciousPackageInsight = &controltowerv1.PublishPackageInsightRequest_MaliciousPackageInsight{
+			AnalysisId: mar.AnalysisId,
+			IsMalware:  mar.IsMalware,
+			IsVerified: mar.VerificationRecord != nil && mar.VerificationRecord.GetIsMalware(),
```

Comment:
`IsVerified` should be independent of if it is malware or not right? We can even verify and mark a package as `NOT MALWARE`. This is particularly useful to deal with false positives. I think we should just check if verification record is available to make a decision on `IsVerified`